spectator/statelessregistry_test.cc

```diff
@@ -61,9 +61,12 @@ TEST(StatelessRegistry, Gauge) {
   TestStatelessRegistry r;
   auto g = r.GetGauge("foo");
   auto g2 = r.GetGauge("bar", {{"id", "2"}});
+  auto g3 = r.GetGaugeTTL("baz", 1, {{"id", "2"}});
```

Comment:
Also show off the case where there are no tags:

```
auto g3 = r.GetGaugeTTL("baz", 1);
auto g34= r.GetGaugeTTL("baz", 2, {{"id", "2"}});
```

The tests for this library serve two purposes: (1) ensuring correctness, and (2) demonstrating usage.